src/Components/WhyWeDidItSection/WhyWeDidItSection.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './WhyWeDidItSection.css';
+import { Typography } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+function WhyWeDidItSection({ imgSrc, title, text }) {
+  return (
+    <div className="align">
+      <img src={imgSrc} className="icon-size" />
```

Comment:
is this an icon? cause the css class name says it's an icon

Reply:
Yes it is, but as u told us yesterday bcz we were facing a problem with the svg file, to download it and insert it as an image, the className is just describing it lol, but in reality it's an img of that icon :)